lib/propolis/src/hw/virtio/pci.rs

```diff
@@ -667,19 +667,25 @@ impl IsrState {
     }
     /// Read ISR value.  Returns (`intr_queue`, `intr_cfg`)
     fn read(&self) -> (bool, bool) {
-        let inner = self.inner.lock().unwrap();
+        let inner = self.0.lock().unwrap();
         (inner.intr_queue, inner.intr_cfg)
     }
-    /// Write ISR value
-    fn write(&self, intr_queue: bool, intr_cfg: bool) {
-        self.sync_pin(|inner| {
-            inner.intr_queue = intr_queue;
-            inner.intr_cfg = intr_cfg;
-        });
+    /// Import ISR value
+    ///
+    /// Sets the internal ISR value without propogating the state to the
```

Comment:
nit: s/propogating/propagating

Reply:
*shakes fist at `set spell`*

Reply:
Fixed, along with another nit I found while I was in the neighborhood